opentelemetry-sdk/tests/trace/test_sampling.py

```diff
@@ -192,40 +199,40 @@ def test_default_off(self):
             None,
             0xDEADBEF1,
             "unsampled parent, sampling off",
+            trace.SpanKind.INTERNAL,
             attributes={"sampled.expect": "false"},
-            trace_state=trace_state,
         )
         self.assertFalse(default_off.decision.is_sampled())
         self.assertEqual(default_off.attributes, {})
-        self.assertEqual(default_off.trace_state, trace_state)
+        self.assertIsNone(default_off.trace_state)
 
     def test_probability_sampler(self):
-        trace_state = trace.TraceState([("key", "value")])
         sampler = sampling.TraceIdRatioBased(0.5)
 
         # Check that we sample based on the trace ID if the parent context is
         # null
+        # trace_state should also be empty since it is based off of parent
         sampled_result = sampler.should_sample(
             None,
             0x7FFFFFFFFFFFFFFF,
             "sampled true",
+            trace.SpanKind.INTERNAL,
             attributes={"sampled.expect": "true"},
-            trace_state=trace_state,
         )
         self.assertTrue(sampled_result.decision.is_sampled())
         self.assertEqual(sampled_result.attributes, {"sampled.expect": "true"})
-        self.assertEqual(sampled_result.trace_state, trace_state)
+        self.assertFalse(sampled_result.trace_state)
```

Comment:
Should it be `assertIsNone` here?
```suggestion
        self.assertIsNone(sampled_result.trace_state)
```